digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -1,11 +1,13 @@
-package fi.liikennevirasto.digiroad2
+package fi.liikennevirasto
```

Comment:
Why are we changing this file package?

Reply:
Geometry utilities methods should be accessible outside digiroad2. Can be reverted if it cause problems